app/models/conversion_host.rb

```diff
@@ -91,7 +94,7 @@ def verify_credentials(auth_type = nil, options = {})
   #  - The number of concurrent tasks has not reached the limit.
   #
   def eligible?
-    source_transport_method.present? && verify_credentials && check_concurrent_tasks
+    source_transport_method.present? && authentication_check('v2v').first && check_concurrent_tasks
```

Comment:
Hm I was assuming that you would call `authentication_status_ok?` here...is anyone calling `verify_credentials` anymore now ?  I don't see it being used in this file.

Reply:
It gets called internally via the `authentication_check` (via a private method within that method).

https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/authentication_mixin.rb#L349

Reply:
Ah okay :+1: